### PR TITLE
Fix some miscellaneous grammar mistakes

### PR DESCRIPTION
**Pull request by SpaceManiac**
Fixes such glories as:
> That's a REACTIVE CHEMICALS.
> That's Skillsoft Station.
> There are 3 hot ices in the stack.
> There are 7 regenerative meshs in the stack.
> There are 12 block of sands in the stack.

### CODE REVIEW

**Comment by actioninja** [comment]
Adhesive medical strips

**Review by Rohesie — APPROVED** [review]
Pretty straightforward

File: code/game/objects/items/stacks/sheets/mineral.dm [hunk 1]
```diff
@@ -485,10 +485,10 @@ GLOBAL_LIST_INIT(metalhydrogen_recipes, list(
 	))
 
 /obj/item/stack/sheet/mineral/metal_hydrogen
-	name = "Metal Hydrogen"
+	name = "metal hydrogen"
```
Comment (Fikou) [inline code/game/objects/items/stacks/sheets/mineral.dm:488]: speaking of metal hydrogen, could you fix up the grammar on metallic hydrogen golems?

**Comment by Ghilker** [comment]
Also thanks for the changes on my stuff! English grammar is not my forte

**Pull request merged**